v2/common/src/main/java/com/google/cloud/teleport/v2/transforms/ConstructGenericRecordsFn.java

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
Lets call this class KeyValueToGenericRecordFn since that is what we are doing here